modules/customize-certificates-api-add-named.adoc

```diff
@@ -18,11 +18,24 @@ client's URL.
 reach the API server.
 * The certificate must have the `subjectAltName` extension for the URL.
 
+
 [WARNING]
 ====
 Do not provide a named certificate for the internal load balancer (host
 name `api-int.<cluster_name>.<base_domain>`). Doing so will leave your
 cluster in a degraded state.
+
+If a certificate chain is used, all certificates must be manually concatenated into a single named certificate file.
```

Comment:
@pbmoses This seems separate from the other text in this warning. This should be moved above to be its own prerequisite bullet, instead of inside this warning block.